src/foam/u2/crunch/CapabilityFeatureView.js

```diff
@@ -180,7 +181,7 @@ foam.CLASS({
         this.crunchService.getJunction(null, this.data.id).then(ucj => {
           if ( ucj && ucj.status === this.CapabilityJunctionStatus.GRANTED ) {
             this.auth.cache = {};
-            this.crunchService.pub('updateJunction');
+            this.crunchService.pub('grantedJunction');
```

Comment:
So this ensures that the store items are reloaded after capabilities are granted - is this correct?